msbuild/Xamarin.iOS.Tasks.Core/Xamarin.iOS.Common.targets

```diff
@@ -724,7 +724,7 @@ Copyright (C) 2013-2016 Xamarin. All rights reserved.
 		</PropertyGroup>
 	</Target>
 
-	<Target Name="_GenerateAppExBundleName" Condition="'$(IsAppExtension)' == 'true'" DependsOnTargets="_ComputeTargetArchitectures;_DetectSigningIdentity">
```

Comment:
Explanation on this bit?

Reply:
@chamons: see the commit: https://github.com/xamarin/xamarin-macios/pull/8708/commits/b1d837d3599f9dcae5b4449c2f3a29871f546d63